lib/Backend.hs

```diff
@@ -20,8 +20,10 @@ import Polysemy.Error                (Error)
 import Polysemy
 import Coffer.Path (EntryPath, Path)
 import BackendName (BackendName)
+import Network.HTTP.Client (Manager)
+import Polysemy.Reader (Reader)
 
-type Effects r = (Member (Embed IO) r, Member (Error CofferError) r)
+type Effects r = (Member (Embed IO) r, Member (Reader (Manager, Manager)) r, Member (Error CofferError) r)
```

Comment:
I would create a proper `ConnectionManagers` record with named fields, instead of a tuple. Otherwise, it's easy to forget which one is the HTTP manager and which one is the HTTPS manager.

Reply:
Done